netmanager/src/views/components/DataDisplay/Map/MapFilter.js

```diff
@@ -18,19 +18,28 @@ import "assets/css/map-filter.css";
 
 const FilterIcon = ({ fill, stroke }) => {
   return (
```

Comment:
I am just wondering if this change is related to the PR description. Isn't this change meant/covered in the map filter PR? Just curious, I could be wrong about the PR's scope.